apps/frontend/src/app/legal/legal/legal.component.html

```diff
@@ -0,0 +1,75 @@
+        notifications about the poll you are participating in. You can opt-out of these notifications at any time.</p>
+    </li>
+    <li>
+      <p><strong>Available times</strong>
+        When participating in a poll, you may be asked to provide your available times. This information is used to
+        determine the best time for an event.</p>
+    </li>
+  </ul>
+  <h3>Legal Basis for Data Processing</h3>
+  <p>Apollusia operates in compliance with the data protection regulations applicable in Germany, where our platform is
+    hosted. The collection and processing of personal information is carried out based on the legal grounds of user
+    consent. When you voluntarily provide your personal information by participating in a poll and utilizing our
+    services, you
+    are providing your explicit consent for us to process the information for the purposes outlined in this Privacy
+    Policy.</p>
+  <h3>Statistics</h3>
+  <p>Apollusia collects certain anonymized statistics to improve our service and better understand user interactions.
+    These statistics include the number of polls created, general usage patterns and trends. Please note that these
+    statistics are aggregated and anonymized, ensuring that individual users cannot be identified through this data.</p>
+  <h3>Cookies</h3>
+  <p>At Apollusia, we utilize cookies to enhance your experience and offer convenient functionality, especially for
+    users accessing our platform on different end devices. Cookies are small text files stored on your device, which
+    contain data such as a unique token. The specific purpose of the cookies we employ is to allow users to seamlessly
+    continue using Apollusia on different end devices. By storing a unique token in these cookies, we can provide
+    you with a consistent experience across sessions. By continuing to use our platform, you consent to the use of
+    cookies as described.</p>
+  <h3>Security Measures</h3>
+  <p>Our website operates over HTTPS (Hypertext Transfer Protocol Secure), which provides a secure and encrypted
+    connection between your browser and our servers. This encryption helps protect the data you exchange with our
+    platform, including any personal information you provide. Furthermore, we regularly update our security measures
+    to ensure that your data is protected. While we strive to maintain the highest level of security, it's important to
+    note that no method of data transmission over the internet is entirely foolproof. Therefore, while we implement
+    robust security measures, we cannot guarantee absolute security of your data.</p>
+  <h3>User Rights</h3>
+  <p>As a user of Apollusia, you have certain rights regarding your personal information, which we are committed to
+    respecting and facilitating:</p>
+  <ul>
+    <li>
+      <p><strong>Access</strong>
+        You have the right to request information about the personal data we hold about you and how it is being used.
```

Comment:
How?

Reply:
Will be added later under the `Contact Us`-section. Since we deploy automatically after a PR I would not open another PR in this case as planned, but use this one to realize the imprint with the .env file in this PR.